src/Npgsql/NpgsqlDataSourceBuilder.cs

```diff
@@ -153,6 +155,26 @@ public NpgsqlDataSourceBuilder UseClientCertificatesCallback(Action<X509Certific
         return this;
     }
 
+    /// <summary>
+    /// Sets the <see cref="X509Certificate2" /> that will be used validate SSL certificate, received from the server.
+    /// </summary>
+    /// <param name="caCertificate">The CA certificate.</param>
+    /// <returns>The same builder instance so that multiple calls can be chained.</returns>
+    public NpgsqlDataSourceBuilder UseCACertificate(X509Certificate2? caCertificate)
```

Comment:
```suggestion
    public NpgsqlDataSourceBuilder UseValidationRootCertificate(X509Certificate2? caCertificate)
```

Naming-wise, the [PG docs](https://www.postgresql.org/docs/current/libpq-ssl.html) seem to refer to this as the "root certificate", rather than the "CA certificate". Also, maybe we should make it clearer that this is specifically for server certificate validation?

If so, rename related names as well

Reply:
> Also, maybe we should make it clearer that this is specifically for server certificate validation?

In the summary it's specified that:

> ...that will be used to validate SSL certificate, received from the server

Seems to be straightforward enough.

Reply:
Up to you.

Reply:
Looking over the naming again, I think we should maybe drop "Validation" and just call it UseRootCertificate (sorry for the back and forth). This is because we already have Settings.RootCertificate, the env var PGSSLROOTCERT, the default file `~/root.crt`. It seems to be just "root certificate" everywhere without the word "validation" (or CA).